unlock_indicator.c

```diff
@@ -6,22 +6,22 @@
  * See LICENSE for licensing information
  *
  */
+#include "unlock_indicator.h"
```

Comment:
I'm guessing that you have a linter or something - can you disable header reordering in it?

Reply:
OK - I will wait with the push until I know what you want me to do regarding the oder comments.